### ISSUE TITLE
Nothing in Home App

### ISSUE THREAD

**Issue by FaboFabo93** [moritzmhmk/homebridge-camera-rpi#6]
Hi, I installed all as described but nothing appeared in my home app. No crash on running of Homebridge process and the camera is running on another port as I supposed. Can you help me?

**Comment by nikriek** [comment]
Have you tried running the standalone mode? This worked for me

**Comment by FaboFabo93** [comment]
That mode is used for the same result?

**Comment by FaboFabo93** [comment]
I tried this and it works! Thank you very much!

**Comment by FaboFabo93** [comment]
Only a problem. Running the ip camera on one process, the home bridge process doesn't work anymore. There is a dns name conflict error.

**Comment by FaboFabo93** [comment]
How can I improve the video quality?

**Comment by moritzmhmk** [comment]
What is wrong with the quality? (lag, resolution, framerate...)

**Comment by charlietomo** [comment]
Hi, I have similarly followed the instructions for the homebridge plugin and everything appears to install except nothing new appears in my home app. I would prefer to get it working without using the standalone version if possible.

What should I look for in logs? In terms of related things I get:
```
Loaded plugin: homebridge-camera-rpi
Registering platform 'homebridge-camera-rpi.rpi-camera'
[rpi-camera] Initializing rpi-camera platform...
Loading 2 accessories...
Pi Camera is running on port 39379.
no did or loc found!
```

I added the following to my config.json
```
        {
            "platform" : "rpi-camera",
            "cameras": [{"name": "Pi Camera"}]
        }

```
But must admit I don't fully understand the naming convention. Thanks in advance.

**Comment by moritzmhmk** [comment]
try changing id, pincode and username

e.g.
```
{
  "platform" : "rpi-camera",
  "cameras": [{
    "name": "Pi Camera",
    "id": "foobar",
    "pincode": "042-45-154",
    "username": "EC:42:3D:D3:CE:CE"
 }]
}
```

**Comment by charlietomo** [comment]
I followed the instructions under "**Installation (as homebridge plugin)**" which makes no mention of id/pincode/username etc?
Did I miss some stages of install, or is that for the standalone version?

**Comment by charlietomo** [comment]
Following up I have got this to work. I did not have to add extra details (but thanks for the input).
I had to, in my Home app on iPhone, click `+` in top right, then `Add Accessory` then `Don't have a code or Can't Scan?` - then I saw **Pi Camera** listed. I added it, using the same code as for my main Homebridge setup (specified earlier, once, in my config.json file).

Can I suggest this info is added to the readme? This may have solved the OPs issue as well.
